sdk/storage/storage-blob/test/perfstress/track-2/storageTest.spec.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { generateUuid } from "@azure/core-http";
+import { PerfStressTest, getEnvVar } from "@azure/test-utils-perfstress";
+import { BlobServiceClient, ContainerClient, StorageSharedKeyCredential } from "../../../src";
+import { getValueInConnString } from "../../../src/utils/utils.common";
+
+export abstract class StorageBlobTest<TOptions> extends PerfStressTest<TOptions> {
+  blobServiceClient: BlobServiceClient;
+  containerClient: ContainerClient;
+  sharedKeyCredential: StorageSharedKeyCredential;
+  static containerName = generateUuid();
+
+  constructor() {
+    super();
+    const connectionString = getEnvVar("STORAGE_CONNECTION_STRING");
+    this.sharedKeyCredential = new StorageSharedKeyCredential(
+      getValueInConnString(connectionString, "AccountName"),
+      getValueInConnString(connectionString, "AccountKey")
+    );
+    this.blobServiceClient = BlobServiceClient.fromConnectionString(connectionString);
+    this.containerClient = this.blobServiceClient.getContainerClient(StorageBlobTest.containerName);
+  }
+
+  public async globalSetup() {
```

Comment:
Call base setup and cleanup

Reply:
This is also the base class(for track-2) that extends `PerfStressTest`, in which `globalSetup` and the other methods aren't implemented. Hence, not doing.

Reply:
You should always call the base methods, even if they are currently a no-op.  It's the contract when overriding a method.  It's possible we will add some logic to the based methods in the future.

Reply:
As per the conclusion from #13125, we are retaining the current design of the framework which means we won't be calling the base methods in PerfStressTest class as they are not defined.